src/components/metadata/form/AVUFormList.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * @author psarando
+ */
+import React from "react";
+
+import { FastField, FieldArray, getIn } from "formik";
+import PropTypes from "prop-types";
+
+import { useTranslation } from "i18n";
+
+import DEDialog from "components/utils/DEDialog";
+
+import ids from "../ids";
+import styles from "../styles";
+
+import MetadataList from "../listing";
+import SlideUpTransition from "../SlideUpTransition";
+
+import { build, FormTextField, getFormError } from "@cyverse-de/ui-lib";
+
+import {
+    Button,
+    Divider,
+    Accordion,
+    AccordionSummary,
+    AccordionDetails,
+    Tooltip,
+    Typography,
+    makeStyles,
+} from "@material-ui/core";
+
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+const useStyles = makeStyles(styles);
+
+const AVUFormDialog = (props) => {
+    const {
+        avu,
+        open,
+        closeAttrDialog,
+        editable,
+        targetName,
+        field,
+        touched,
+        errors,
+    } = props;
+
+    const { attr, avus } = avu;
+
+    const [editingAttrIndex, setEditingAttrIndex] = React.useState(-1);
+
+    const { t } = useTranslation("metadata");
+    const classes = useStyles();
+
+    const avuErrors = getFormError(field, touched, errors);
+    const error = avuErrors && avuErrors.error;
+
+    const formID = build(ids.EDIT_METADATA_FORM, field, ids.DIALOG);
+
+    const title = t(
+        editable ? "dialogTitleEditMetadataFor" : "dialogTitleViewMetadataFor",
+        { targetName }
+    );
+
+    const hasChildren = avus && avus.length > 0;
+
+    return (
+        <DEDialog
+            baseId={formID}
+            open={open}
+            title={title}
+            onClose={() => !error && closeAttrDialog()}
+            disableBackdropClick
+            disableEscapeKeyDown
+            TransitionComponent={SlideUpTransition}
+            actions={
+                <Tooltip
+                    title={error ? t("errAVUEditFormTooltip") : ""}
+                    placement="left-start"
+                    enterDelay={200}
+                >
+                    <span>
+                        <Button
+                            id={build(formID, ids.BUTTONS.DONE)}
+                            color="primary"
+                            variant="contained"
+                            disabled={!!error}
```

Comment:
I think we moved away from `disabling` button on validation errors.

Reply:
Good catch. I can update this to show an error announcer instead.